internal/saucecloud/cloud.go

```diff
@@ -259,3 +259,18 @@ func (r *CloudRunner) validateTunnel(id string) error {
 	log.Info().Msg("Tunnel is ready!")
 	return nil
 }
+
+func (r *CloudRunner) dryRun(project interface{}, files []string, sauceIgnoreFile string, suiteNames string) error {
+	tmpDir, err := ioutil.TempDir("./", "sauce-app-payload-*")
```

Comment:
`ioutil` is deprecated. Would be good to use `io` or `os` directly, whichever is more appropriate :)